src/olympia/blocklist/tasks.py

```diff
@@ -10,3 +25,60 @@ def create_blocks_from_multi_block(multi_block_submit_id, **kw):
     obj = MultiBlockSubmit.objects.get(pk=multi_block_submit_id)
     # create the blocks from the guids in the multi_block
     obj.save_to_blocks()
+
+
+@task
+@use_primary_db
+def import_block_from_blocklist(record):
+    log.debug('Processing block id: [%s]', record.get('id'))
+    guid = record.get('guid')
+    if not guid:
+        log.error('GUID is falsey, skipping.')
```

Comment:
Is this supposed to happen? If not, shouldn't we specify the `id` in this log message so that it is easier to debug if we need to.

Reply:
None of the blocklist records currently on kinto have a empty guid - and new blocks require a guid afaik - so it's it's very unlikely to happen.  I was counting on the previous log entry being `'Processing block id: [%s]', record.get('id')` so it could be traced from there in the worst case.

Reply:
> I was counting on the previous log entry being `'Processing block id: [%s]', record.get('id')` so it could be traced from there in the worst case.

How do you guarantee the exact order? The command fires a new task for each record.

Reply:
well, it's 3 lines above in the same task so... 😄